### PR TITLE
fix: My collection bug fixes

### PR DESCRIPTION
**Pull request by MounirDhahri**
The type of this PR is: **Bugfix**

This PR resolves [CX-1983]
This PR resolves [CX-1982]
This PR resolves [CX-1981]

### Description
Group 2 (4)

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

- My Collection Copy Updates - mounir
- Remove submit button from my collection - mounir

#### Android user-facing changes

-

#### Dev changes

- Support superscript in StickyTabPage - mounir

</details>

[CX-1983]: https://artsyproduct.atlassian.net/browse/CX-1983
[CX-1982]: https://artsyproduct.atlassian.net/browse/CX-1982
[CX-1981]: https://artsyproduct.atlassian.net/browse/CX-1981

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Dev changes (Support superscript in StickyTabPage - mounir)
- iOS user-facing changes (My Collection Copy Updates - mounir,Remove submit button from my collection - mounir)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 65a11332ff87d083ef578ee5171067300625704b
</p>

**Review by lordkiz — APPROVED** [review]
Looks good to me 👍

File: src/lib/Scenes/MyProfile/MyCollectionAndSavedWorks.tsx [hunk 1]
```diff
@@ -28,6 +28,7 @@ export const MyCollectionAndSavedWorks: React.FC<Props> = ({ initialTab = Tab.co
           title: Tab.collection,
           content: <MyCollectionQueryRenderer />,
           initial: initialTab === Tab.collection,
+          superscript: "Beta",
```
Comment (brainbicycle) [inline src/lib/Scenes/MyProfile/MyCollectionAndSavedWorks.tsx:31]: 🥼 🔬

**Review by brainbicycle — APPROVED** [review]
Thanks Mounir!

**Pull request merged**